vernac/obligations.ml

```diff
@@ -20,6 +20,12 @@ open Pp
 open CErrors
 open Util
 
+(* EJGA: This should go away, no more need for fix_exn  *)
+module Stm = struct
+    let u = (fun x -> x)
+    let get_fix_exn () = u
+end
+
```

Comment:
If I'm right you need to craft a Program definition that, for example, fails the termination check.
Or something like that, ie. an error only spot by the kernel.